src/samplers/number.js

```diff
@@ -1,31 +1,30 @@
-const faker = require('faker');
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 export function sampleNumber(schema) {
-  let res;
+  const MAX_VALUE = schema.type && schema.type === 'integer' ?
+    Number.MAX_SAFE_INTEGER : Number.MAX_VALUE;
+  const MIN_VALUE = Number.MIN_SAFE_INTEGER;
 
-  if (schema.maximum && schema.minimum) {
-    res = schema.exclusiveMinimum ? Math.floor(schema.minimum) + 1 : schema.minimum;
-    if (schema.exclusiveMaximum && res >= schema.maximum || !schema.exclusiveMaximum && res > schema.maximum) {
-      res = (schema.maximum + schema.minimum) / 2;
-    }
-    return res;
-  }
+  let schemaMin = schema.minimum + !!schema.exclusiveMinimum;
+  let schemaMax = schema.maximum - !!schema.exclusiveMaximum;
```

Comment:
Do you specifically write code so that no one else really got it? 🤣

Replace with:
```javascript
let schemaMin = schema.minimum && schema.exclusiveMinimum ? 
    schema.minimum + 1 : 
    schema.minimum;
  let schemaMax = schema.maximum && schema.exclusiveMaximum ? 
    schema.maximum - 1 : 
    schema.maximum;
```

Reply:
Aw cmon its simply its either 0 or 1 xD

Reply:
Hahaha) You implicit cast boolean to number after then you explicit cast boolean to boolean. Is it easier?